Makefile

```diff
@@ -6,12 +6,7 @@ include .travis/docker.mk
 validate: test.requirements test
 
 test: clean
-	./manage.py test --settings=notesserver.settings.test --with-coverage --with-ignore-docstrings \
-		--exclude-dir=notesserver/settings --cover-inclusive --cover-branches \
```

Comment:
We should probably update `.coveragrc` to still calculate branch coverage; see the `branch` option under https://coverage.readthedocs.io/en/coverage-5.0.3/config.html#run

Reply:
done